test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+
+describe RecipesController do
+  describe 'index' do
+    it "can get the index path" do
+      VCR.use_cassette("recipes") do
+        get recipes_path, params: {message: "chicken"}
```

Comment:
What if the user provides a search term that yields no results, like `dkafjlaslhalkg`?